AutomatedTesting/Gem/PythonTests/editor/EditorScripts/ComponentCRUD_Add_Delete_Components.py

```diff
@@ -114,7 +115,7 @@ async def add_component(component_name):
         Report.result(Tests.mesh_component_added, hydra.has_components(entity_id, ['Mesh']))
 
         # 6) Delete Mesh Component
-        general.idle_wait(0.5)
+        general.idle_wait_frames(3)     # Wait for Inspector to refresh
```

Comment:
similar here, is there something specific that should appear in the inspector that signals it has loaded what was expected?